src/dotnet/APIView/ClientSPA/src/app/_components/revisions-list/revisions-list.component.ts

```diff
@@ -506,7 +507,7 @@ export class RevisionsListComponent implements OnInit, OnChanges {
           `Run <code>dotnet pack</code>`, 
           `Upload the resulting .nupkg file.`
         ];
-        this.acceptedFilesForReviewUpload = ".nupkg";
+        this.acceptedFilesForReviewUpload = ".nupkg, .dll";
```

Comment:
so this is a current bug in prod for manual uploads?

Reply:
This is weird. let me do a rebase. Main is fine

Reply:
weird. it is still showing up here